examples/kubernetes/1.9/cilium.yaml

```diff
@@ -221,7 +226,6 @@ spec:
           secret:
             secretName: cilium-etcd-secrets
       restartPolicy: Always
-      priorityClassName: system-node-critical
```

Comment:
Is this intended to be removed?

Reply:
Yes, `priorityClassName` was only introduced in 1.11.

Reply:
I don't think that's true. PriorityClass existed as alpha since 1.8. 1.11 marked it as beta.

Reply:
alpha features are not enabled by default in kubernetes, only beta. To prevent avoiding possible kubernetes errors we should not have options that are not enabled by default in clusters.